frontend/src/context/layers/admin_level_data.ts

```diff
@@ -59,45 +71,76 @@ export async function getAdminLevelDataLayerData(
     throw new Error('Boundary Layer not loaded!');
   }
   const adminBoundaries = adminBoundariesLayer.data;
+  const adminBoundaryFeatureProps = adminBoundaries.features.map(feature =>
+    pick(feature.properties, [
+      adminBoundaryLayer.adminCode,
+      ...adminBoundaryLayer.adminLevelNames,
+      ...adminBoundaryLayer.adminLevelLocalNames,
+    ]),
+  );
+  const layerData: DataRecord[] = compact(
+    adminBoundaryFeatureProps.map(adminBoundaryFeatureProp => {
```

Comment:
Let's add some comments to explain the different steps.

Should this be a reduce instead to avoid the `eslint-disable-next-line fp/no-mutation` ?

Reply:
since the fallback value is conditionally set, I don't believe the reduce will help avoid the `eslint disable`